### DISCUSSION TITLE
Added: Notes from July 6, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#3656] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-07-06.md

Agenda
* Repeated attributes on partial members
* `sealed` on `data` members
* Required properties

**Comment by HaloFour** [comment]
Any thoughts on how "initialization debt" might work in combination with "init methods"?  Like could an "init method" be considered to set more than one required member in one call?

**Comment by 333fred** [comment]
>  Like could an "init method" be considered to set more than one required member in one call?

I would expect they could. We'll need more design work here.
